test/functional/rpc_blockchain.py

```
@@ -247,6 +248,14 @@ def _test_getblockchaininfo(self):
                 'active': False},
         })
 
+    def _test_y2106(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_test_y2106`

Add return type annotation: `None`

(ANN202)